src/platform/platform.h

```diff
@@ -38,4 +40,31 @@
  */
 void PLATFORM_init(void);
 
+typedef enum {
+    TIMER1,
+    TIMER2,
+    TIMER3,
+    TIMER4,
+    TIMER5,
+    TIMER6,
+    TIMER7,
+    TIMER8,
+    TIMER16,
+    TIMER17
+} PeripheralClock;
```

Comment:
Public types should be prefixed with the module name in caps. Also, I think this type or its members are misnamed. A timer is not a clock; it *has* a clock.


```suggestion
typedef enum {
    PLATFORM_CLOCK_TIMER1,
    PLATFORM_CLOCK_TIMER2,
    PLATFORM_CLOCK_TIMER3,
    PLATFORM_CLOCK_TIMER4,
    PLATFORM_CLOCK_TIMER5,
    PLATFORM_CLOCK_TIMER6,
    PLATFORM_CLOCK_TIMER7,
    PLATFORM_CLOCK_TIMER8,
    PLATFORM_CLOCK_TIMER16,
    PLATFORM_CLOCK_TIMER17
} PLATFORM_PeripheralClock;
```

Reply:
Sure, and then e.g. PLATFORM_CLOCK_GPIO4 is the right name, right?

Reply:
Yes!
I had missed the CLOCK prefix, have made that change and pushed it as well